app/src/main/java/com/infomaniak/drive/utils/AccountUtils.kt

```diff
@@ -301,4 +315,8 @@ object AccountUtils : CredentialManager {
     }
 
     fun isEnableAppSync(): Boolean = UploadFile.getAppSyncSettings() != null
+
+    fun initInfomaniakLogin(context: Context): InfomaniakLogin {
+        return InfomaniakLogin(context, appUID = BuildConfig.APPLICATION_ID, clientID = BuildConfig.CLIENT_ID)
+    }
```

Comment:
It should be in a `companion object`.
And be called `getInfomaniakLogin`.
It's not really an init

And maybe even an extension of Context

Reply:
Done (moved in  Extension.kt)